keyboards/satan/keymaps/olligranlund_iso/keymap.c

```diff
@@ -48,6 +48,9 @@
 #define CALC KC_CALC  // Open default calculator app
 #define MYCM KC_MYCM  // Open default file manager
 
+// increase readability
+#define XXXXX    KC_NO
```

Comment:
Is this required when `XXXXXXX` already exists?

```suggestion
```

Reply:
Well, it's not predefined, like `XXXXXXX`, so it depends on what you want.

Ideally, I think we should use the built in one.

Reply:
I would also prefer use of the seven-character alias defined in QMK core.

Reply:
@OlliGranlund I recommend you rewrite this keymap to use the `LAYOUT_60_iso` macro. (Your existing readme suggests that's the layout you're running anyway.) Right now this keymap is based on outdated conventions, and wouldn't compile from a current version of QMK if merged as-is.